src/services/logger.ts

```diff
@@ -1,17 +1,17 @@
-import winston from "winston";
+import winston from 'winston';
 
-const { combine, timestamp, printf } = winston.format
+const { combine, timestamp, printf } = winston.format;
 
 const logger = winston.createLogger({
-    level: "info",
-    format: combine(
-        timestamp(),
-        printf(info => `${info.timestamp} ${info.level}: ${info.message}`)
-    ),
-    transports: [
-        new winston.transports.File({ filename: "logs/error.log", level: "error" }),
-        new winston.transports.File({ filename: "logs/combined.log" })
-    ]
-})
+  level: 'info',
```

Comment:
Create a .env and a .env.example file. Do not commit the .env file. Add these

```
PORT=8000
LOG_LEVEL=debug
```
Log level should default to debug during development. Then fetch this level in the logger from the .env file

```suggestion
  level: process.env.LOG_LEVEL,
```